src/theory/quantifiers/quantifiers_rewriter.cpp

```diff
@@ -1672,8 +1672,11 @@ Node QuantifiersRewriter::computeMiniscoping( std::vector< Node >& args, Node bo
     }
     return mkForAll( newArgs, body[1], qa );
   }else if( body.getKind()==AND ){
-    if( options::miniscopeQuant() ){
-      //break apart
+    if (options::miniscopeQuant() || options::aggressiveMiniscopeQuant())
+    {
+      // aggressive miniscoping implies that structural miniscoping should
+      // be applied first
```

Comment:
move this to before the `if`? The concatenation of this with the below comment is slightly confusing